sdk/python/tests/integration/materialization/test_offline_online_store_consistency.py

```diff
@@ -332,7 +332,7 @@ def check_offline_and_online_features(
 def run_offline_online_store_consistency_test(
     fs: FeatureStore, fv: FeatureView, full_feature_names: bool,
 ) -> None:
-    now = datetime.utcnow()
+    now = datetime.now()
```

Comment:
same q

Reply:
same response. create_data_set was always using utcnow, but there are other tests that don't use utcnow and instead use now(). This unifies across our test suites